addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1759,7 +1775,20 @@ var SnippetEditor = Widget.extend({
                 this.dragState.currentHeight = (maxRowEnd - 1) * (gridProp.rowSize + gridProp.rowGap) - gridProp.rowGap;
             }
         }
-    }
+    },
+    /**
+     * Called when the "replace" overlay button is clicked.
+     *
+     * @private
+     * @param {Event} ev
+     */
+    _onReplaceClick(ev) {
+        ev.stopPropagation();
+        this.trigger_up('open_add_snippet_dialog', {
+            snippetGroup: undefined,
```

Comment:
No need to set `snippetGroup: undefined` (it's `null` by default in `_openAddSnippetDialog`).